package.json

```diff
@@ -67,6 +71,9 @@
   "babel": {
     "presets": [
       "es2015"
+    ],
+    "plugins": [
+      "transform-object-rest-spread"
```

Comment:
o.O uh oh ... I didn't keep up for some it seems, didn't know it was already in stage 2.